src/App/Utilities/AppHelpers.cs

```diff
@@ -593,5 +594,18 @@ public static async Task ClearServiceCacheAsync()
             searchService.ClearIndex();
             usernameGenerationService.ClearCache();
         }
+
+        public static async Task BiometricsTooManyAttempts(bool hasMasterPassword, bool pinEnabled)
+        {
+            if (hasMasterPassword || pinEnabled)
+            {
+                return;
+            }
+
+            var platformUtilsService = ServiceContainer.Resolve<IPlatformUtilsService>();
+            var vaultTimeoutService = ServiceContainer.Resolve<IVaultTimeoutService>();
+            await platformUtilsService.ShowDialogAsync(AppResources.BiometricsLoginExceeded, AppResources.Warning, AppResources.OkGotIt);
+            await vaultTimeoutService.LogOutAsync();
+        }
```

Comment:
🎨 I think this should be moved to the `IBiometricsService` (to the base class) so we don't make this helper class bigger, which IMO should have several methods moved to services. Doing this, we try to concentrate everything relating to biometrics in the same place.
Edit: However, we could move it to `MobilePlatformUtilsService`, either the entire code or just a call to the `IBiometricsService` that performs this. Because as far as I see the same action is passed from the callers. Additionally, I'd change the parameters to something that is more specific to the method like `bool shouldLogOutOnBiometricsAttemptsExceeded` and then you can just put the `or` condition on the caller.
What do you think?